Feature_files/gmail.feature

```diff
@@ -0,0 +1,7 @@
+Feature: verify facebook page
+  @start
+  Scenario: Verify user is able to launch facebook page
+    Given I am on facebook page
+    When I should see and fill the login fields
```

Comment:
Here, when is the action. You can rewrite or update like below

When I send valid login credentials
Then I am able to login successfully